alicloud/resource_alicloud_cs_kubernetes.go

```diff
@@ -88,12 +99,12 @@ func resourceAlicloudCSKubernetes() *schema.Resource {
 			},
 			"master_instance_types": &schema.Schema{
 				Type:     schema.TypeList,
-				Required: true,
+				Optional: true,
```

Comment:
we have three types of kubernetes cluster, singleAz/multiAz/managed.
master_instance_types is only required by singleAz and multiAz, not need for managed.
this argument's validation is still there in `isMultiAZClusterAndCheck()`
the new change has been documented in the help docs.